system-test/Dockerfile.linux

```diff
@@ -7,18 +7,17 @@ RUN go get github.com/google/pprof
 
 FROM debian:stretch
 
-RUN apt-get update && apt-get install -y \
-    curl \
-    python \
-    make \
-    g++ \
- && rm -rf /var/lib/apt/lists/*
+ARG NODE_VERSION
+ARG NVM_NODEJS_ORG_MIRROR
+ARG BUILD_BINARIES
+
+RUN apt-get update && apt-get install -y curl
+RUN if [ "$BUILD_BINARIES" = "true" ] ; then apt-get install -y python g++ make ; fi
```

Comment:
Do this for Dockerfile.node-alpine files too?

Reply:
Done.